webapps/ui/cockpit/src/plugins/cockpit.processInstance.runtime.tab/variables/Variables.js

```diff
@@ -0,0 +1,278 @@
+      return await (
+        await post(
+          `%ENGINE_API%/variable-instance/count`,
+          Object.assign(
+            {
+              processInstanceIdIn: [id]
+            },
+            PAYLOAD,
+            search
+          )
+        )
+      ).json();
+    };
+
+    return fetchJobsCount();
+  }, [id, search]);
+
+  const handleSortChange = property => {
+    const sortObj = {
+      sortBy: property,
+      sortOrder:
+        sortOrder.sortBy === property
+          ? sortOrder.sortOrder === "desc"
+            ? "asc"
+            : "desc"
+          : "asc"
+    };
+    setItem("sortHistVarInstTab", sortObj);
+    setSortOrder(sortObj);
+  };
+
+  const getSearchQuery = activityInstanceId => {
+    if (activityInstanceId) {
+      return encodeURI(
+        JSON.stringify([
+          {
+            type: "activityInstanceIdIn",
+            operator: "eq",
+            name: "",
+            value: activityInstanceId
+          }
+        ])
+      );
+    }
+  };
+
+  const getActivityInstance = activityInstanceId => {
+    if (instanceIdToInstancesMap) {
+      const activityInstance = instanceIdToInstancesMap[activityInstanceId];
+      if (activityInstance) {
+        return activityInstance;
+      } else {
+        return activityInstances;
+      }
+    }
+  };
+
+  const getScope = activityInstanceId => {
+    const activityInstance = getActivityInstance(activityInstanceId);
+    return activityInstance?.activityName
+      ? activityInstance?.activityName
+      : activityInstance.name
+      ? activityInstance.name
+      : getActivityName(bpmnElements[definition?.key]);
+  };
+
+  const handleSearch = useCallback(payload => setSearch(payload), []);
+
+  return (
+    <>
+      <div className="VariablesTab">
+        <SearchPills
+          rowsCount={rowsCount}
+          criteria={searchPillsConfig.criteria}
+          labels={searchPillsConfig.labels}
+          onChange={handleSearch}
+        />
+
+        {search && (
+          <TablePaginated
+            rowsPerPage={ROWS_PER_PAGE}
+            header={header.map((head, idx) => {
+              return (
+                <>
+                  {head.searchQuery && (
+                    <Table.Head
+                      key={idx}
+                      onSort={() => handleSortChange(head.searchQuery)}
+                      sortOrder={
+                        sortOrder.sortBy === head.searchQuery
+                          ? sortOrder.sortOrder
+                          : null
+                      }
+                    >
+                      {translate(head.label)}
+                    </Table.Head>
+                  )}
+                  {!head.searchQuery && (
+                    <Table.Head key={idx}>{translate(head.label)}</Table.Head>
+                  )}
+                </>
+              );
+            })}
+            generateRow={variable => (
+              <>
+                <Table.Cell>
```

Comment:
The Fragment contains mutliple `Table.Cell` children. This currently creates a warning because they don't have unique keys